xt/lib/PageObject/App/AP/Invoice.pm

```diff
@@ -41,6 +41,7 @@ sub select_vendor {
     $elem->send_keys($vendor);
 
     $self->update;
+    $self->session->page->body->maindiv->wait_for_content(replaces => $elem);
```

Comment:
Nice find!

(However, I think it makes more sense to do this here: https://github.com/ledgersmb/LedgerSMB/blob/4d02bed8491eb6b4244a09b6244541a7e65d360c/xt/lib/PageObject/App/Invoice.pm#L27)